io/build.sbt

```diff
@@ -1,8 +1,11 @@
 import Dependencies._
 
 libraryDependencies ++= Seq(
-  AwsJavaSdkS3,
   AwsJavaSdkCore,
+  AwsJavaSdkS3,
+  // FIX: Explicitly override transient bouncy castle versions from `sshj`: https://security.snyk.io/vuln/SNYK-JAVA-ORGBOUNCYCASTLE-6612984
```

Comment:
Can you leave a link to [this PR](https://github.com/hierynomus/sshj/pull/938) stating that we can remove the explicit evictions once a version of sshj with that patch is released?

Reply:
Sure! https://github.com/adzerk/apso/compare/5a118b8abedd7a86a875f4bdcaa5f89e64976eee..7c1530732554553bc561dc51efeedb9537b3c566

> Thanks! I'll prepare a new release once this is merged in.

Thank you 🙏